src/draw/nxp/vglite/lv_draw_vglite.c

```diff
@@ -282,9 +288,26 @@ static int32_t _vglite_dispatch(lv_draw_unit_t * draw_unit, lv_layer_t * layer)
     /* Try to get an ready to draw. */
     lv_draw_task_t * t = lv_draw_get_next_available_task(layer, NULL, DRAW_UNIT_ID_VGLITE);
 
-    if(t == NULL || t->preferred_draw_unit_id != DRAW_UNIT_ID_VGLITE)
+    if(t == NULL)
         return LV_DRAW_UNIT_IDLE;
 
+    if(lv_draw_get_unit_count() > 1) {
+        /* Let the SW unit to draw this task. */
+        if(t->preferred_draw_unit_id != DRAW_UNIT_ID_VGLITE)
+            return LV_DRAW_UNIT_IDLE;
+    }
+    else {
+        /* Fake unsupported tasks as ready. */
+        if(t->preferred_draw_unit_id != DRAW_UNIT_ID_VGLITE) {
+            t->state = LV_DRAW_TASK_STATE_READY;
+
+            /* Request a new dispatching as it can get a new task. */
+            lv_draw_dispatch_request();
```

Comment:
Having a `while` loop here to get as many tasks are possible might be faster. (Not critical now)